src/content/en/2022/mobile-web.md

```diff
@@ -2,21 +2,436 @@
+
+The `mailto` protocol is found on 29% of mobile pages. Close behind is the `tel` protocol, found on 27% of pages. That these protocols are found more on mobile pages is potentially indicative that websites are being modified with functionality to encourage a more multi-modal experience on mobile, like placing a call, which would be more complex to execute on desktop.
+
+{{ figure_markup(
+    caption="Percent of mobile pages that use the `sms` link protocol.",
+    content="0.27%",
+    classes="big-number",
+    sheets_gid="1551447761",
+    sql_file="popular_link_protocols.sql"
+  )
+}}
+
+Adoption of the `sms` protocol on mobile websites is much lower, coming in at 0.27%. This is interesting because linking someone to a pre-formatted SMS message can be an effective way to encourage people to sign up for an SMS loyalty program that can keep users much more engaged with a brand in the long-term, but it seems likely that designers, developers, or marketers do not consider this when they are building a site.
+
+Messaging application links are used very rarely by comparison, at less than 1% adoption. This could potentially be explained by the growth of _deep linking_ on the mobile web, whereby an app can be directly opened and navigated to a deep section within the app.
+
+
+### Input fields
+
+Engagement and interactivity are critical for a good mobile web experience, but for years many developers have overlooked setting specific type declarations for mobile users, to ensure that the correct keyboard appears immediately, when the user finally decides to interact. The correct setting will return a number keyboard when a user is entering a phone number, and a keyboard that includes an `@` symbol when they are entering an email address. Other types of input fields include submit functions, searches, checkboxes, password fields, radio buttons, other buttons, or as hidden elements.
+
+{{ figure_markup(
+    image="input-types.png",
+    caption="Adoption of input types.",
+    description="Bar chart showing adoption of input types, with text on 71% of pages, hidden on 53%, submit on 38%, email on 27%, unknown (or N/A) on 27%, checkbox on 27%, search on 25%, password on 13%, tel on 6%, radio on 6%, and button on 5%.",
+    chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ2iu1w8TLfOTLCQ1O-d74ku9NirGCQJNeO789uKX1-M6N053ASsz2tXWPvU1twhScsa3rxWF7_7OBn/pubchart?oid=1619844270&format=interactive",
+    sheets_gid="938177161",
+    sql_file="popular_mobile_input_types.sql"
+  )
+}}
+
+Since modulating keyboards with input fields is much more important on mobile than it is on desktop, we track the most popular mobile input types across their archive of websites. In 2022, `text` was still the most popular input type, used on 71% of pages, which is down slightly from [last year](https://almanac.httparchive.org/en/2021/mobile-web#type-declarations) at 73%. `hidden` was the next most common input type at 53% of pages, then `submit` at 38%.
```

Comment:
```suggestion
Since modulating keyboards with input fields is much more important on mobile than it is on desktop, we track the most popular mobile input types across their archive of websites. In 2022, `text` was still the most popular input type, used on 71% of pages, which is down slightly from [last year](../2021/mobile-web#type-declarations) at 73%. `hidden` was the next most common input type at 53% of pages, then `submit` at 38%.
```